webpack.config.cjs

```diff
@@ -270,9 +270,7 @@ const webStoriesScripts = {
   },
   plugins: [
     ...sharedConfig.plugins,
-    new DependencyExtractionWebpackPlugin({
-      injectPolyfill: true,
-    }),
+    new DependencyExtractionWebpackPlugin(),
```

Comment:
Nicpic. This is the only place here this is different https://github.com/google/web-stories-wp/blob/8fbe2baca71b362d97b810de0661360329daa579/webpack.config.cjs#L325-L327

I think we should `new DependencyExtractionWebpackPlugin(),` part of `sharedConfig` object and use a filter to change it for editor / dashboard. This would remove lots of repeated code.

Reply:
I feel like using something like 

```js
plugins: [
...sharedConfig.plugins.filter(
  (plugin) =>
    !(plugin instanceof DependencyExtractionWebpackPlugin)
),
new DependencyExtractionWebpackPlugin({ 
  requestToExternal, 
}),
]
```

makes the config less readable there. Looks a bit confusing.

Reply:
It is a little less readable, but without it, doesn't it mean spinning up a new instance of `DependencyExtractionWebpackPlugin` like 5 times? Also it is messy in one place, instead of lots of repeated code...